projects/sandbox/src/app/_models/tiers.ts

```diff
@@ -18,10 +18,10 @@ export enum DisplayedSubsection {
 export interface DatasetTierSummaryBase {
```

Comment:
I've renamed `DatasetTierSummaryBase` to `TierSummaryBase`, because it represents not just record tier data, but also aggregations of floored data for the entire dataset (as well as any user selection).

I've renamed `DatasetTierSummaryRecord` to `TierSummaryRecord`, and `DatasetTierSummary` has been deleted, because it wasn't even needed!